definitions/procedures/repositories/validate.rb

```diff
@@ -0,0 +1,26 @@
+module Procedures::Repositories
+  class Validate < ForemanMaintain::Procedure
+    metadata do
+      description 'Validate repositories'
+
+      confine do
+        feature(:downstream) || feature(:upstream)
+      end
+
+      param :version,
+            'Version for which repositories needs to be validated',
+            :required => true
+    end
+
+    def run
+      with_spinner("Validating repositories for #{@version}") do
+        absent_repos = (feature(:downstream) || feature(:upstream)).absent_repos(@version)
```

Comment:
Since we don't define this for upstream, I think this should stay downstream only.

Reply:
Noted and done.